### PR TITLE
stkGHO Emissions Restart

### PR DESCRIPTION
**Pull request by efecarranza**
# Changelog

Add 90 days distribution to AAVE emissions for stkGHO.
Update allowance.

### CODE REVIEW

File: src/20240424_AaveV3Ethereum_StkGHOIncentives/AaveV3Ethereum_StkGHOIncentives_20240424.sol [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      AaveV3EthereumAssets.AAVE_UNDERLYING,
+      AaveSafetyModule.STK_GHO,
+      0
+    );
+
+    MiscEthereum.AAVE_ECOSYSTEM_RESERVE_CONTROLLER.approve(
+      MiscEthereum.ECOSYSTEM_RESERVE,
+      AaveV3EthereumAssets.AAVE_UNDERLYING,
+      AaveSafetyModule.STK_GHO,
+      AAVE_EMISSION_PER_SECOND_STK_GHO * DISTRIBUTION_DURATION
```
Comment (brotherlymite) [inline src/20240424_AaveV3Ethereum_StkGHOIncentives/AaveV3Ethereum_StkGHOIncentives_20240424.sol:38]: @efecarranza previous allowance should also be added along with the current value to approve.

Comment (Luigy-Lemon) [reply src/20240424_AaveV3Ethereum_StkGHOIncentives/AaveV3Ethereum_StkGHOIncentives_20240424.sol]: Included in the latest commit. Thanks

File: src/20240424_AaveV3Ethereum_StkGHOIncentives/AaveV3Ethereum_StkGHOIncentives_20240424.sol [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    MiscEthereum.AAVE_ECOSYSTEM_RESERVE_CONTROLLER.approve(
+      MiscEthereum.ECOSYSTEM_RESERVE,
+      AaveV3EthereumAssets.AAVE_UNDERLYING,
+      AaveSafetyModule.STK_GHO,
+      remainingAllowance + (AAVE_EMISSION_PER_SECOND_STK_GHO * DISTRIBUTION_DURATION)
+    );
+
+    // Allowance of 700 AAVE to Merit to compensate missed stkGHO rewards
+    MiscEthereum.AAVE_ECOSYSTEM_RESERVE_CONTROLLER.approve(
```
Comment (brotherlymite) [inline src/20240424_AaveV3Ethereum_StkGHOIncentives/AaveV3Ethereum_StkGHOIncentives_20240424.sol:57]: @efecarranza @Luigy-Lemon Could we remove the allowance to the multi-sig, as it betters fits on the phase-2 proposal that would be created at a later stage

Comment (efecarranza) [reply src/20240424_AaveV3Ethereum_StkGHOIncentives/AaveV3Ethereum_StkGHOIncentives_20240424.sol]: done

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**